### PR TITLE
Replace PaymentInstrumentValueObject with inline definitions

### PR DESCRIPTION
**Pull request by swilczynski**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Transactions/Transaction.yaml [hunk 1]
```diff
@@ -123,7 +123,13 @@ allOf:
         format: bin
         readOnly: true
       paymentInstrument:
-        $ref: ../PaymentInstruments/ValueObjects/PaymentInstrumentValueObject.yaml
+        type: object
+        description: Default payment instrument information.
```
Comment (ashkarpetin) [inline openapi/components/schemas/Transactions/Transaction.yaml:127]:
```suggestion
        description: Payment instrument information.
```

Comment (swilczynski) [reply openapi/components/schemas/Transactions/Transaction.yaml]: Thanks for noticing

File: openapi/components/schemas/Customer.yaml [hunk 2]
```diff
@@ -43,8 +43,12 @@ properties:
   defaultPaymentInstrument:
     type: object
     nullable: true
-    allOf:
-      - $ref: ./PaymentInstruments/ValueObjects/PaymentInstrumentValueObject.yaml
+    description: Default payment instrument information.
```
Comment (nanobash) [inline openapi/components/schemas/Customer.yaml:46]: Perhaps here should be the same description then?

```suggestion
    description: Payment instrument information.
```

**Review by ashkarpetin — APPROVED** [review]

File: openapi/components/schemas/Customer.yaml (hunk 2 above)
Comment (ashkarpetin) [reply openapi/components/schemas/Customer.yaml:46]: Customer has default payment instrument

**Review by nanobash — APPROVED** [review]

**Pull request merged**